aitelemetry/telemetrywrapper.go

```diff
@@ -90,6 +92,14 @@ func NewAITelemetry(
 	id string,
 	aiConfig AIConfig,
 ) TelemetryHandle {
+	env, _ := os.LookupEnv("AZACN_TESTENV")
```

Comment:
why does it say TESTENV ?

Reply:
is it supposed to say AZACN_CLOUDENV ? or just ENV ?

Reply:
this is for uni test..just for making sure not conflicting with any other env variable

Reply:
No what I mean is why does the variable name says TESTENV. I thought it's a test environment - which is not the case here. I think it can be a bit misleading

Reply:
this is for unit test to pass...unit test can't make call to metadata server..let me check if i can do with mockagent by passing url